app/models.py

```
@@ -428,6 +435,26 @@ def get_last_odometer(self, distance_unit=None):
+        Returns the highest from trips or charging sessions.
+        """
+        # If Tessie is enabled, use Tessie battery level exclusively
+        if self.uses_tessie_battery() and self.tessie_battery_level:
+            return round(self.tessie_battery_level)
+
+        last_trip = self.trips.filter(Trip.end_fuel_level.isnot(None)).order_by(Trip.date.desc()).first()
+
+        last_charge = self.charging_sessions.filter(ChargingSession.end_soc.isnot(None)).order_by(
+            ChargingSession.date.desc()).first()
+
+        if not last_trip and not last_charge: return 0
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)